docs/application/dotnet/guides/user-interface/nui/text.md

```diff
@@ -756,6 +756,108 @@ The following table lists the available `TextEditor` properties:
 | `TranslatableText`                 | String      | Specifies the `TranslatableText` property that sets the SID value. |
 
 
+## How to use clipboard APIs with select text
```

Comment:
@shrouqsabah Hello. Hope you are well!

After checking before merging, I have a query regarding this PR.

In the heading, you have mentioned `SelectedText`, but in your changes here, you have written `SelectText` API. From what I have observed, `SelectText` is the appropriate name, as mentioned [here](https://docs.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextField.html#Tizen_NUI_BaseComponents_TextField_SelectText_System_Int32_System_Int32_). Can you please confirm if it should be `SelectedText` API or `SelectText` API?

Reply:
@safir-srbd , Hello, I hope you well too.
Yes, you are right. It should be ```SelectText```. 
I will update title.

Reply:
@shrouqsabah Thank you for your response!

In that case, I have a heading suggestion for this section which would be more aligned with the rest of the titles in this document. 
What do you think of "Use clipboard and SelectText APIs" as the heading?
If it seems ok, please update it and we can merge this PR.